rskj-core/src/test/java/co/rsk/peg/BridgeTest.java

```diff
@@ -640,10 +642,135 @@ public void getBtcBlockchainBestChainHeightOnlyAllowsLocalCalls_beforeRskip220()
         Assert.assertTrue(bridge.getBtcBlockchainBestChainHeightOnlyAllowsLocalCalls(new Object[0]));
     }
 
+    @Test
+    public void activeAndRetiringFederationOnly_activeFederationIsNotFromFederateMember_retiringFederationIsNull_throwsVMException() throws Exception {
+        // Given
+        BridgeMethods.BridgeMethodExecutor executor = Bridge.activeAndRetiringFederationOnly(
+                null,
+                null
+        );
+
+        int senderPK = 999; // Sender PK does not belong to Member PKs
+        Integer[] memberPKs = new Integer[]{100, 200, 300, 400, 500, 600};
```

Comment:
Why doesn't it make sense to have two methods in `FederationTestUtils` like `getActiveFederation` and `getRetiringFederation?` (talking about the class as a whole, not this specific test).
I ask because I see that when we ask for an active federation, we always send the (100, 200, 300, 400, 500, 600) PKs (same thing with the retiring federation).
This way, maybe we can ignore declaring the PKs here (since the test doesn't care about them).

Reply:
In this particular case it makes sense to have them as an array to improve readability (since I need for the sender PK to be in relationship with the array of pks). I think that if we want to change it as you point, we would like to improve the documentation (javadoc) of the "getActive..." and "getRetiring..." methods. But I think it could be possible!

Reply:
@fedejinich @Vovchyk what do you think?

Reply:
I personally think that in this case I will leave it as it is to improve visibility... Maybe we can revisit this in a future, I know that @fedejinich is thinking in updating how this tests are executed, so maybe we can tackle this right there.

Reply:
agree with you naza!